### ISSUE TITLE
Upload-finish response

### ISSUE THREAD

**Issue by chevinbrown** [JonRowe/phoenix_live_view_dropzone#12]
This is awesome!

I'm using cloudinary for uploads--I get a response like this: 
```
{
 public_id: 'sample',
 version: '1312461204',
 width: 864,
 height: 564,
 format: 'jpg',
 created_at: '2017-08-10T09:55:32Z',
 resource_type: 'image',
 tags: [], 
 bytes: 9597, 
 type: 'upload', 
 etag: 'd1ac0ee70a9a36b14887aca7f7211737', 
 url: 'http://res.cloudinary.com/demo/image/upload/v1312461204/sample.jpg',
 secure_url: 'https://res.cloudinary.com/demo/image/upload/v1312461204/sample.jpg',
 signature: 'abcdefgc024acceb1c1baa8dca46717137fa5ae0c3',
 original_filename: 'sample'
}
```
Those details I need to persist, and AFAIK, using this lib doesn't allow me to retrieve the asset b/c the data (mainly the id) is determined by the external resource. 

I'd love to help, but my elm is pretty rusty right now. 😃

**Comment by JonRowe** [comment]
Yeah the library currently doesn't do anything with the response from the server, as I've never encountered one that needs it, the id is entirely for keeping track of which file is which with the live view, it should be possible to emit the response from the file upload back to the live view though.

**Comment by rynam0** [comment]
I would love this as well... Same use-case but with google cloud storage. I'm needing to persist the returned mediaUrl, etc. I have zero experience with Elm so probably unable to help out here.

**Comment by JonRowe** [comment]
Try using the `response-body` branch, I threw together the Elm but haven't had time to test it with the component to check it works to release it.

**Comment by rynam0** [comment]
Thanks, @JonRowe!  I tried out the branch and now see a "body" element in the `file-status` map.  I feel like a total schmuck right now as I neglected to notice that GCS does not return a response body!  Hopefully this will help @chevinbrown though!  Thanks so much for the speedy response and a simple solution to liveview uploads!

**Comment by chevinbrown** [comment]
@rynam0 it certainly may, although LV-upload is in master now 😂...I'll try to get back to testing this & if it works I'll let Jon know; I think it certainly would make sense to include as part of the lib.

**Comment by rynam0** [comment]
I did see that it was merged but I haven't seen any docs or know how to use it yet.  I too think it makes sense to include this. Hopefully it works with your cloudinary example!

**Comment by JonRowe** [comment]
The response-body branch was released as 0.11
